types/styled-components/styled-components-tests.tsx

```diff
@@ -4,14 +4,16 @@ import * as ReactDOMServer from 'react-dom/server';
 
 import styled, {
     css,
-    injectGlobal,
+    createGlobalStyle,
     isStyledComponent,
     keyframes,
     ServerStyleSheet,
     StyleSheetManager,
     ThemeProps,
     ThemeProvider,
     withTheme,
+    ThemeConsumer,
+    ThemeConsumerComponent,
```

Comment:
What's this? We don't have an export called "ThemeConsumerComponent"

Reply:
It's a type export, not a component. I accidentally left it in the imports even though it's unused. I'll be removing it from there shortly.